### PR TITLE
change the parameter for private net and upload the ui for DID function

### PR DESCRIPTION
**Pull request by 286545761**

### CODE REVIEW

File: ELA/Base/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -642,7 +640,85 @@
...
+"err20051"="Invalid local store";
+"err20052"="Master wallet is not exist";
+"err20053"="Asset not found";
+"err20054"="Read config file error";
+"err20055"="Invalid chain id";
+"err20056"="Unsupport old transaction";
+"err20057"="Unsupport operation";
+"err20058"="Big int error";
+"err20059"="Deposit utxo not found";
+//"err20060"="账户UTXO数量过多, 须先进行零钱换整";
```
Comment (Zero0000O) [inline ELA/Base/en.lproj/Localizable.strings:719]: Is this unused in the English environment?

File: ELA/MeVC/cell/HWMShowSocialAccountTableViewCell.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//
+//  HWMShowSocialAccountTableViewCell.h
+//  elastos wallet
+//
+//  Created by 韩铭文 on 2020/2/4.
+//
```
Comment (Zero0000O) [inline ELA/MeVC/cell/HWMShowSocialAccountTableViewCell.h:6]: The copyright information needs to be modified.

**Comment by 286545761** [comment]
This is the author is not copyright information the za is not the author information?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "BochengZhang"<notifications@github.com&gt;; 
发送时间: 2020年2月25日(星期二) 晚上6:16
收件人: "elastos/Elastos.App.UnionSquare.iOS"<Elastos.App.UnionSquare.iOS@noreply.github.com&gt;; 
抄送: "人生如梦"<286545761@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [elastos/Elastos.App.UnionSquare.iOS] Dev (#1)




@bocheng0000 commented on this pull request.
 
 
In ELA/MeVC/cell/HWMShowSocialAccountTableViewCell.h:
 &gt; @@ -0,0 +1,18 @@ +// +//  HWMShowSocialAccountTableViewCell.h +//  elastos wallet +// +//  Created by 韩铭文 on 2020/2/4. +//  
The copyright information needs to be modified.

**Comment by bocheng0000** [comment]
Yes, there should not be any author information too.

**Comment by Zero0000O** [comment]
Yes, there should not be any author information too.

**Comment by 286545761** [comment]
ok



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "BochengZhang"<notifications@github.com&gt;; 
发送时间: 2020年2月25日(星期二) 晚上7:27
收件人: "elastos/Elastos.App.UnionSquare.iOS"<Elastos.App.UnionSquare.iOS@noreply.github.com&gt;; 
抄送: "人生如梦"<286545761@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [elastos/Elastos.App.UnionSquare.iOS] change the parameter for private net and upload the ui for DID function (#1)




Yes, there should not be any author information too.

**Review by heropan — APPROVED** [review]

**Comment by Zero0000O** [comment]
Please modify the copyright and author information in the later pr.

**Pull request merged**